lib/utils/tar.js

```diff
@@ -28,8 +28,8 @@ const logTar = (tarball, opts = {}) => {
     { name: 'name:', value: tarball.name },
     { name: 'version:', value: tarball.version },
     tarball.filename && { name: 'filename:', value: tarball.filename },
-    { name: 'package size:', value: byteSize(tarball.size) },
-    { name: 'unpacked size:', value: byteSize(tarball.unpackedSize) },
+    { name: 'package size:', value: formatBytes(tarball.size) },
```

Comment:
I don't think these are actually used anywhere because byteSize would be returning an object here not a string.